api/migrations/schema/Version20240817112552.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240817112552 extends AbstractMigration {
+    public function getDescription(): string {
+        return 'Rename camp.name to shortTitle and switch shortTitle with title if shortTitle is longer than 16 characters and title is shorter than shortTitle';
+    }
+
+    public function up(Schema $schema): void {
+        $this->addSql('ALTER TABLE camp RENAME COLUMN name TO shortTitle');
+        $this->addSql('ALTER TABLE camp ALTER shortTitle DROP NOT NULL, ALTER shortTitle TYPE TEXT');
+        $this->addSql('UPDATE camp SET shortTitle = null WHERE shortTitle = title');
+        $this->addSql('UPDATE camp SET shortTitle = title, title = shortTitle WHERE char_length(shortTitle) > 16 AND char_length(title) < char_length(shortTitle);');
+        // $this->addSql('UPDATE camp SET shortTitle = left(shortTitle, 16) WHERE char_length(shortTitle) > 16;');
```

Comment:
i would delete this line.
If you want to save it for later, keep it in your local notes.

But the migration works, i tested it against a current prod backup